README.md

```diff
@@ -297,7 +297,11 @@ return {
 {
   root = vim.fn.stdpath("data") .. "/lazy", -- directory where plugins will be installed
   defaults = {
-    lazy = false, -- should plugins be lazy-loaded?
+    -- Enables lazy loading by default for plugins without triggers,
+    -- see the "Lazy Loading" documentation section for more information.
+    -- Note, this does not have same meaning as lazy = false in the plugin spec,
+    -- which can disable lazy loading completely.
```

Comment:
```suggestion
```
Imo it should already be clear that the value in the specs overrides this when you look at the referred section

Reply:
That section does not mention anything about `spec.lazy` at all. It only talks about the `module`

The spec documentation also does not mention that you can disable the lazy loading by setting it to false. 

So perhaps those two sections need updating as well.